mypyc/ir/ops.py

```diff
@@ -1372,6 +1373,28 @@ def accept(self, visitor: 'OpVisitor[T]') -> T:
         return visitor.visit_load_mem(self)
 
 
+class GetElementPtr(RegisterOp):
+    """Get the address of a struct element"""
+    error_kind = ERR_NEVER
+
+    def __init__(self, src: Value, src_type: RType, index: int, line: int = -1) -> None:
```

Comment:
It would be more user friendly to take the name of the struct field instead of the index.